### PR TITLE
Feature/sc 8214/app add user s email and full name to profile

### PR DESCRIPTION
**Pull request by lvancraen**
### Why

Ticket speaks for itself. It's simply adding the full name and email of the user in the profile page.

I took some time to refactor the page into TS as well.

Screen Shot 2022-09-22 at 11 54 31 AM

### Ticket

[https://app.shortcut.com/capable/story/<id>](https://app.shortcut.com/capable/story/8214/app-add-user-s-email-and-full-name-to-profile-page)

### Accomplishments

- [x] refactored some code into TS
- [x] added the full name and email to the profile page

### CODE REVIEW

File: src/pages/Profile.tsx [hunk 1]
```diff
@@ -100,9 +103,9 @@ const ProfileAvatar = ({ patient, setError, setPatient }) => {
   );
 };
 
-const usePatient = () => {
+const usePatient = (): [Patient, SetPatient, boolean] => {
```
Comment (ysael) [inline src/pages/Profile.tsx:106]: would be nice to more that hook to react-query at one point like we do in Kariko, I also think we should incrementally move away from -> swr as we now have react-query that does the same

**Review by ysael — APPROVED** [review]
Minor suggestion that can be done in this pr or later

**Comment by ysael** [comment]
Andreea looks different :)

File: src/pages/Profile.tsx (hunk 1 above)
Comment (lvancraen) [reply src/pages/Profile.tsx:106]: I'll do it in a subsequent PR if that's ok. I know there's a rush to get this out.

**Pull request merged**